lib/CMakeLists.txt

```diff
@@ -91,19 +95,16 @@ if (SWIFT_SWIFT_PARSER)
             "${SWIFT_HOST_LIBRARIES_DEST_DIR}/${module_dir}/${module_file}")
     endforeach()
 
-    add_custom_command(
-      OUTPUT ${dest_module_files}
-      COMMAND ${CMAKE_COMMAND} -E make_directory ${SWIFT_HOST_LIBRARIES_DEST_DIR}/${module_dir}
-      COMMAND ${CMAKE_COMMAND} -E copy_if_different ${module_files} ${SWIFT_HOST_LIBRARIES_DEST_DIR}/${module_dir}/
-      COMMENT "Copying ${module_dir}"
-    )
+    add_custom_target(copy_swiftSyntaxLibrary_${module_dir}
+                      DEPENDS ${module_files}
+                      COMMAND ${CMAKE_COMMAND} -E make_directory ${SWIFT_HOST_LIBRARIES_DEST_DIR}/${module_dir}
+                      COMMAND ${CMAKE_COMMAND} -E copy_if_different ${module_files} ${SWIFT_HOST_LIBRARIES_DEST_DIR}/${module_dir}/
+                      COMMENT "Copying ${module_dir}"
+                     )
+    add_dependencies(swiftSyntaxLibraries copy_swiftSyntaxLibrary_${module_dir})
 
     list(APPEND SWIFT_SYNTAX_DEST_FILES ${dest_module_files})
```

Comment:
We can remove this now, it was only used to add the dependency.

Reply:
Done